ExampleApp/firstTest_BurstDebugInformation_DoNotShip/tempburstlibs/armeabi-v7a/lib_burst_generated.txt

```diff
@@ -0,0 +1,166 @@
+Library: lib_burst_generated
```

Comment:
can we check if this could/should be added to gitignore

Reply:
there is "do not ship" in the name of the directory, so I deleted it from the repo and added it to gitignore

Reply:
i disabled burst compilation, now it wont produce them anyway